multiply_ui/ui/__init__.py

```diff
@@ -1,9 +1,3 @@
 from .mui import mui
 
-# TODO: remove following imports
-
-# noinspection PyUnresolvedReferences
-from .data import data_ui
```

Comment:
With these changes, Notebooks "multiply-ui-1.ipynb" and "multiply-ui-2.ipynb" have become obsolete. Please remove them, at the very least from "readme.txt"